### PR TITLE
Fix #899: Add warning for wrong URL format

### LINKED ISSUE #899: Throw warnings if developer forgets the `url( )` wrapper for srcs.
For properties of type `src` (propertyTypes.js), if developer does ```srcProp: myimage.png``` instead of ```srcProp: url(myimage.png)```, throw a console warning. This can be detected if the querySelector fails and there isn't a ```#``` at the beginning of the string.

https://github.com/aframevr/aframe/blob/master/src/core/propertyTypes.js#L117

### PR DESCRIPTION
**Pull request by darkwing**
**Description:**
Fix for #899 

**Changes proposed:**
- Simply add warning for it
- Added a few tests for giggles.

### CODE REVIEW

**Comment by darkwing** [comment]
On PTO today (ill); will add test for selector as soon as I get a moment.

**Comment by darkwing** [comment]
Updated

File: src/core/propertyTypes.js [hunk 1]
```diff
@@ -117,6 +118,10 @@ function srcParse (value) {
   var el = selectorParse(value);
   if (el) { return el.getAttribute('src'); }
 
+  if (value.charAt(0) !== '#') {
```
Comment (ngokevin) [inline src/core/propertyTypes.js]: Selectors can be a class or attribute or data selector.

Comment (ngokevin) [inline src/core/propertyTypes.js]: Or any element name

Comment (darkwing) [inline src/core/propertyTypes.js]: Right; I was following your logic here:  https://github.com/aframevr/aframe/issues/899

I'd assumed you made ID selection a standard.  Is your `#` comment no longer true?

Comment (ngokevin) [inline src/core/propertyTypes.js]: Sorry, you're right. I was thinking selector not src

File: src/core/propertyTypes.js [hunk 2]
```diff
@@ -117,6 +118,10 @@ function srcParse (value) {
...
+    warn('Value provided for `src` attribute is not valid: ' + value + '. Value must be selector notation or wrapped in `url()`.');
```
Comment (ngokevin) [inline src/core/propertyTypes.js]: Should we mention this is either ID or `url()`, rather than just "selector notation"?

Comment (darkwing) [inline src/core/propertyTypes.js]: Yes, I'll update the language this morning.

**Comment by darkwing** [comment]
Updated with mores specific message.

**Pull request merged**